### PR TITLE
Advise newcomers to work on branch

### PR DESCRIPTION
**Pull request by Mariatta**
### Checklist

- [x] I have rebased properly. Please see [our tutorial on rebasing](http://coala.readthedocs.io/en/latest/Developers/Git_Basics.html#rebasing).
- [x] I have gone through the [commit guidelines](http://coala.readthedocs.io/en/latest/Developers/Writing_Good_Commits.html) and I've followed them.
- [x] I have followed the [guidelines for the commit shortlog](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#shortlog).
- [x] I have followed the [guidelines for the commit body](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#commit-body).
- [x] I have included the issue URL with the 'Fixes' keyword if the commit fixes a *real bug* and the 'Closes' keyword if it adds a feature or enhancement. For details, check the [issue reference section in our commit guidelines](http://docs.coala.io/en/latest/Developers/Writing_Good_Commits.html#issue-reference).
- [x] I have [run all the tests](http://coala.readthedocs.io/en/latest/Developers/Executing_Tests.html) and they all pass. If any CI (below) is not green, please fix them. If GitMate issues appear you will have to amend your commits, pushing new commits on top is not sufficient!

### Reviewers

Mention that newcomers should not work on master branch.
Add instruction on how to create a branch.

Closes https://github.com/coala/coala/issues/3148

### CODE REVIEW

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -146,7 +154,22 @@ Step 5. Sending Your Changes
 ----------------------------
 
 Now that you've fixed the issue, you've tested it and you think it is ready
-to be merged, create a commit and push it to your fork, using:
+to be committed.
```
Comment (Mixih) [inline docs/Developers/Newcomers_Guide.rst]: Hmm, I might just be too picky here, but this sentence seems to dangle. Perhaps you could terminate it (by adding "you should check your branch") or by moving the note earlier or later.

**Comment by Mariatta** [comment]
Thanks @mixih :) updated based on your comment.

**Comment by Mixih** [comment]
looks good. I'm acking. Feel free to merge after the circle passes

**Comment by Mixih** [comment]
ack 3bb6e2e

**Comment by Mariatta** [comment]
Thanks :)

**Comment by Mariatta** [comment]
@rultor merge

**Pull request merged**